Controller/ContentManagement/JobController.php

```diff
@@ -56,8 +56,17 @@ public function jobStatusAction(Job $job)
         $theme = new Theme();
         $converter = new AnsiToHtmlConverter($theme);
 
+        $status = preg_replace_callback(
```

Comment:
Logic should not be part of the controller. Can this be done in a data object?

Reply:
This is a hack which one detects url within the last line of the job ouput. This is a view stuff but I wont take care of it in the twig and I don' t need it anywhere else (nor create a twig fucntion). But ideally the AnsiToHmltConverter should support the [Console Hyperlink](https://symfony.com/blog/new-in-symfony-4-3-console-hyperlinks)

Reply:
Okay but this is technical debt. I would prefer a correct implementation because it’s not clear what is happening (=will waste dev time if someone passes this function)

Reply:
refactored in the common